### PR TITLE
P0 product listing

### PR DESCRIPTION
**Pull request by purvasharmadev**
<h3>Minor changes </h3>
<li>Removed extra spaces between code.</li>
<li>Added Links</li>

<h3>Main Changes</h3>
<li>Added the product listing page with filter section</li>

kstyle-product-page

### CODE REVIEW

File: product/product.css [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+  .card-header .category{
+      color:var(--TEXT-COLOR);
+      position: absolute;
+      bottom: 1.5rem;
+      left: 1rem;
+      background-color:var(--SECONDARY-COLOR);
+      padding: 3px;
+      border-radius:0px 5px 5px 0px;
+      font-size: 14px;
```
Comment (zoef-yt) [inline product/product.css:216]: Use rem

File: product/product.css [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    border-radius: 50%;
+    top: -1px;
+    left: -1px;
+    background-color: var(--PRIMARY-COLOR);
+}
+
+input[type=radio]:checked:after {
+    content: "";
+    display: block;
+    border: 2px solid white;
```
Comment (zoef-yt) [inline product/product.css:164]: Use var for colours

File: product/product.css [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+input[type=radio]:before {
+    content: "";
+    display: block;
+    position: absolute;
+    width: 1.25rem;
+    height: 1.25rem;
+    border-radius: 50%;
+    top: -1px;
+    left: -1px;
+    background-color: #e9e9e9;
```
Comment (zoef-yt) [inline product/product.css:145]: Use var for colours

File: product/product.css [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+    left: 0;
+    background-color: var(--PRIMARY-COLOR);
+}
+
+input[type=checkbox]:checked:after {
+    content: "";
+    display: block;
+    width: 5px;
+    height: 10px;
+    border: solid white;
```
Comment (zoef-yt) [inline product/product.css:118]: Use var for colours

File: product/product.css [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+/* Display for checkbox */
+input[type=checkbox]:before {
+    content: "";
+    display: block;
+    position: absolute;
+    width: 1.25rem;
+    height: 1.25rem;
+    top: 0;
+    left: 0;
+    background-color: #e9e9e9;
```
Comment (zoef-yt) [inline product/product.css:98]: Use var for colour

File: product/product.css [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+/* Slider Track psuedo class */
+input[type=range]::-webkit-slider-runnable-track {
+    width: 100%;
+    height: 5px;
+    cursor: pointer;
+    transition: 0.2s;
+    box-shadow: 0px 0px 0px #000000;
+    background: var(--SECONDARY-COLOR);
+    border-radius: 10px;
+    border: 0px solid #000000;
```
Comment (zoef-yt) [inline product/product.css:57]: Use var for colour

File: product/product.css [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+    outline: none;
+}
+
+/* Slider Track psuedo class */
+input[type=range]::-webkit-slider-runnable-track {
+    width: 100%;
+    height: 5px;
+    cursor: pointer;
+    transition: 0.2s;
+    box-shadow: 0px 0px 0px #000000;
```
Comment (zoef-yt) [inline product/product.css:54]: Use var for colour

File: product/product.css [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+
+/* Slider */
+.sliderContainer {
+    color: gray;
+    margin-top: 1rem;
+    width: 100%;
+}
+
+.sliderContainer h3 {
+    color: black;
```
Comment (zoef-yt) [inline product/product.css:23]: Use var for colour

File: product/product.css [hunk 9]
```diff
@@ -0,0 +1,221 @@
...
+    height: auto;
+    z-index: 1;
+    overflow: auto;
+    width: 20%;
+}
+
+
+/* Slider */
+.sliderContainer {
+    color: gray;
```
Comment (zoef-yt) [inline product/product.css:17]: Use var for colour

File: index.css [hunk 10]
```diff
@@ -20,11 +23,11 @@ nav{
...
     position: fixed;
     top: 0;
-    z-index:1;
+    z-index: 1;
 }
 
-header{
-    background-image:linear-gradient(45deg,var(--SECONDARY-COLOR),rgba(0, 0, 0, 0.178) ), url("https://variety.com/wp-content/uploads/2021/05/BTS_Butter-photo.jpg");
+header {
+    background-image: linear-gradient(45deg, var(--SECONDARY-COLOR), rgba(0, 0, 0, 0.178)), url("https://variety.com/wp-content/uploads/2021/05/BTS_Butter-photo.jpg");
```
Comment (zoef-yt) [inline index.css:30]: Use var for colour

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: product/product.css [hunk 11]
```diff
@@ -0,0 +1,221 @@
...
+    margin-top: 1rem;
+    width: 100%;
+}
+
+.sliderContainer h3 {
```
Comment (itsmebipul00) [inline product/product.css:22]: variable could be used

File: product/product.css (hunk 5 above)
Comment (itsmebipul00) [inline product/product.css:98]: Same here

File: product/product.css (hunk 3 above)
Comment (itsmebipul00) [inline product/product.css:145]: Here too

**Review by itsmebipul00 — APPROVED** [review]
Nice Work!!

File: product/product.css (hunk 1 above)
Comment (purvasharmadev) [reply product/product.css:216]: we can use px for less than 1rem i.e 16px, right?

File: product/product.css (hunk 2 above)
Comment (purvasharmadev) [reply product/product.css:164]: these colors are not to be used more than 2 times in code so declaring them as a variable is not helpful, I think. please correct me if I am wrong.

**Comment by purvasharmadev** [comment]
> Nice Work!!

Thank you for your time.

File: product/product.css (hunk 1 above)
Comment (zoef-yt) [reply product/product.css:216]: Ohk you haven't done 62.5% thing got it.

**Review by zoef-yt — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Nice code structure, css could be improved more

**Pull request merged**